CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+# How to contribute to FreeOrion
+
+As open source project FreeOrion both relies on contributions from both long
+time developers and one time occasion contributors.  This guide should help you
```

Comment:
As an open source project FreeOrion relies on contributions from both long time developers and one time contributors.

Add "an".  Remove "both". Remove "occasion"